test/types/index.ts

```diff
@@ -256,11 +275,11 @@ server.ready(function (err) {
   if (err) throw err
 })
 
-server.ready(function (err, done) {
+server.ready(function (err: Error, done: Function) {
   done(err)
 })
 
-server.ready(function (err, context, done) {
+server.ready(function (err: Error, context: fastify.FastifyInstance<http2.Http2SecureServer, http2.Http2ServerRequest, http2.Http2ServerResponse>, done: Function) {
```

Comment:
Is it necessary to specify that `done` is a function? I assume the types are added here to demonstrate passing a specific context i.e HTTP2?

Reply:
The types are being added so that tests pass in strict mode. So it can either be type `any` or `Function`